File: 프로그래머스/unrated/181939. 더 크게 합치기/더 크게 합치기.cpp
#include <string>
#include <vector>

using namespace std;

int solution(int a, int b) {
    int answer = 0;
    string abStr = to_string(a)+to_string(b); 
    string baStr = to_string(b)+to_string(a); 
    int ab = stoi(abStr); 
    int ba = stoi(baStr); 
    answer = ab > ba ? ab: ba; 
    return answer;
}

File: 프로그래머스/0/181939. 더 크게 합치기/더 크게 합치기.java
class Solution {
    public int solution(int a, int b) {
        int answer = 0;
        String abstr = Integer.toString(a) + Integer.toString(b); 
        String bastr = Integer.toString(b) + Integer.toString(a); 
        
        answer = Math.max(Integer.parseInt(abstr), Integer.parseInt(bastr));
        return answer;
    }
}